clients/term/client_term.c

```diff
@@ -434,6 +435,14 @@ void editorProcessKeypress() {
 	}
 }
 
+void SiganlHandler(int SignalCaught)
```

Comment:
1. Stick to the coding style. Curly braces on same line as function name, camel case for function names.
2. Correct the function name.

Something like -
```
void signalHandler(int signal) {
    ...
}
```